code/datums/ruins/station.dm

```diff
@@ -1,21 +1,21 @@
-/datum/map_template/ruin/station
-	prefix = "_maps/RandomRuins/StationRuins/"
-	cost = 0
-
-// Boxstation
-	// Engine
-
-/datum/map_template/ruin/station/box/engine
-	id = "engine_sm"
-	suffix = "Box/Engine/engine_sm.dmm"
-	name = "Engine SM"
-
-/datum/map_template/ruin/station/box/engine/singulo
-	id = "engine_singulo"
-	suffix = "Box/Engine/engine_singulo.dmm"
-	name = "Engine Singulo"
-
-/datum/map_template/ruin/station/box/engine/tesla
-	id = "engine_tesla"
-	suffix = "Box/Engine/engine_tesla.dmm"
+/datum/map_template/ruin/station
+	prefix = "_maps/RandomRuins/StationRuins/"
+	cost = 0
+
+// Boxstation
+	// Engine
+
+/datum/map_template/ruin/station/box/engine
+	id = "engine_sm"
+	suffix = "Box/Engine/engine_sm.dmm"
+	name = "Engine SM"
+
+/datum/map_template/ruin/station/box/engine/singulo
+	id = "engine_singulo"
+	suffix = "Box/Engine/engine_singulo.dmm"
+	name = "Engine Singulo"
+
+/datum/map_template/ruin/station/box/engine/tesla
+	id = "engine_tesla"
+	suffix = "Box/Engine/engine_tesla.dmm"
```

Comment:
whitespace changes, most likely line endings. :pouting_man:

Reply:
I didnt touch this file idk what happend to it